src/content/docs/dev/send-tokens/interchain-tokens/no-code.mdx

```diff
@@ -7,11 +7,11 @@
   interactive form through the wizard.
 </Callout>
 
-# No-Code (ITS Portal)
+# No Code Setup
```

Comment:
Do we not want mention of the portal? that's the terminology we typcially use for this so i think there should be some mention in the docs

Reply:
The first paragraph explains that. I think it's redundant.

If you also think about it, no code setup means portal